test/language/expressions/arrow-function/arrow/bind_1.js

```diff
@@ -0,0 +1,17 @@
+// Copyright 2015 Microsoft Corporation. All rights reserved.
+// This code is governed by the license found in the LICENSE file.
+
+/*---
+description: invoke arrow function using bind
+es6id: 14.2
+---*/
+
+var a = (x) => x;
+
+assert.sameValue(a.bind(null, 1)(), 1, "Invocation of arrow function failed using fn.bind(null, 1).");
+
+assert.sameValue(a.bind(undefined, 1)(), 1, "Invocation of arrow function failed using fn.bind(undefined, 1).");
+
+assert.sameValue(a.bind(this, 1)(), 1, "Invocation of arrow function failed using fn.bind(this, 1).");
+
+assert.sameValue(a.bind({}, 1)(), 1, "Invocation of arrow function failed using fn.bind({}, 1).");

```

Comment:
Duplicates

- https://github.com/tc39/test262/blob/master/test/language/expressions/arrow-function/lexical-this.js